.github/.cspell.json

```diff
@@ -0,0 +1,42 @@
+{
+  "ignorePaths": [
+    "**/node_modules/**",
+    "**/vscode-extension/**",
+    "**/.git/**",
+    ".vscode",
+    "megalinter",
+    "package-lock.json",
+    "report"
+  ],
+  "language": "en",
+  "noConfigSearch": true,
+  "words": [
+    "aaaaaaaaaaaaaaaaaaaagaaaaaaaaaaaaaaaaaaa",
```

Comment:
is there a reason to for this, as well as line 39?

Reply:
In the tests, both xaaaaaa and aaaaaa were used, so they were added to this local dictionary